client/lib/importer/actions.js

```diff
@@ -129,14 +129,12 @@ export function finishUpload( importerId, importerStatus ) {
 	} );
 }
 
-export function mapAuthor( importerId, sourceAuthor, targetAuthor ) {
-	Dispatcher.handleViewAction( {
-		type: IMPORTS_AUTHORS_SET_MAPPING,
-		importerId,
-		sourceAuthor,
-		targetAuthor
-	} );
-}
+export const mapAuthor = ( importerId, sourceAuthor, targetAuthor ) => ( {
```

Comment:
I checked several state subfolders looking for actions tests. It looks like in most cases people cover with tests only async actions. So probably it's enough to leave it as it is until we have reducer in place.